README.md

```diff
@@ -4,11 +4,11 @@ Turn a fresh Ubuntu installation into a fully-configured, beautiful, and modern
 
-But package management is only half the battle of getting a great development experience going on Linux. The other half lies in the dotfiles that control the configuration. Linux gets great power from how customizable it is, but that also presents a paradox of choice and a tall learning curve. Having good, curated defaults that integrate all the many tools in a coherent feel and look can help more developers acquire a taste for Linux, which they may then later inspire a fully bespoke setup (or not!).
+But package management is only half the battle of getting a great development experience going on Linux. The other half lies in the dotfiles that control the configuration. Linux gets great power from how customizable it is, but that also presents a paradox of choice and a tall learning curve. Having good, curated defaults that integrate all the many tools in a coherent feel and look can help more developers acquire a taste for Linux, which may then later inspire a fully bespoke setup (or not!).
 
-Nothing in Omakub provides solutions to problems you couldn't also solve a million other ways. The main benefit is in The Omakase Spirit. The idea that an entire setup experience can benefit from being tailored upfront by someone with strong opinions about what works and looks good together. This doesn't make the choices necessarily better than other choices. Linux has inspired a million options for a million tastes. That's great and worthy of celebration. But there's a large constituency of developers who are more than willing to trade ultimate bespoke customization for a cohesive package of goods, at least until they understand what all the options are and have fully bought into making the switch to Linux.
+Nothing in Omakub provides solutions to problems you couldn't also solve a million other ways. The main benefit is in The Omakase Spirit—the idea that an entire setup experience can benefit from being tailored upfront by someone with strong opinions about what works and looks good together. This doesn't make the choices necessarily better than other choices. Linux has inspired a million options for a million tastes. That's great and worthy of celebration. But there's a large constituency of developers who are more than willing to trade ultimate bespoke customization for a cohesive package of goods, at least until they understand what all the options are and have fully bought into making the switch to Linux.
```

Comment:
Purposeful period. Keep original.

Reply:
Understood. My first read through I got tripped up by the structural sentence that isn't a grammatical sentence, but I am one for a stylistic flourish here and there, so happy to oblige.